api/types/src/convert.rs

```diff
@@ -912,15 +912,15 @@ impl<'a, R: MoveResolver + ?Sized> MoveConverter<'a, R> {
             .map(bcs::to_bytes)
             .collect::<Result<_, bcs::Error>>()?;
 
-        Ok(EntryFunction::new(
-            module.into(),
-            function.name.into(),
-            type_arguments
+        Ok(ViewFunction {
```

Comment:
Why this new type? It looks the same as EntryFunction.

Reply:
Because, from a code generation standpoint, names that do different things, but are teh same are confusing.

Is an entry function a view function?  No.  Can either ever be the other? No.

So, even if today it's the same exact form, it could have differences in the future.